internal/liteconfig/config.go

```diff
@@ -93,6 +94,7 @@ func NewDefaultConfig() (*Config, error) {
 		})),
 		portProvider: &portProvider{},
 		FrontendIP:   "",
+		BaseConfig:   &config.Config{},
```

Comment:
Not required, but we could add a unit test to ensure that BaseConfig is used when computing the final config. There are a few other test cases I'd like to add though, so this might be best in a separate PR.